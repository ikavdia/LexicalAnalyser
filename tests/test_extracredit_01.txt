1.asdf
123ab12x16ABC12x16