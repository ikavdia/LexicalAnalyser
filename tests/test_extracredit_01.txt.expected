{1 , NUM , 1}
{ , DOT , 1}
{asdf , ID , 1}
{123 , NUM , 2}
{ab12x16ABC12x16 , ID , 2}
{ , END_OF_FILE , 2}